torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -423,7 +423,9 @@ def gen_check(handle_kind, idx, name, tensor):
                     from torch.utils._sympy.value_ranges import bound_sympy
 
                     sym_range = bound_sympy(d, V.graph.sizevars.shape_env.var_to_range)
-                    if not math.isinf(sym_range.lower):
+                    if config.aot_inductor.check_lowerbound and not math.isinf(
+                        sym_range.lower
+                    ):
```

Comment:
Does it make sense to relax the lower bound to 0 if `sym_range.lower` is 2? Using a config here still can trip many users.